src/js/rtc_session.js

```diff
@@ -484,6 +484,38 @@ export default class RtcSession {
     get mediaStream() {
         return this._userAudioStream;
     }
+    pauseLocalVideo() {
+        if (this._streamToBeClosed) {
```

Comment:
also need to take care of the case that client provides stream rather than connect-rtc-js calls gUM.